coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt

```diff
@@ -75,7 +75,7 @@ fun executeCommand(
 }
 
 private val lineaRollupAddressPattern = Pattern.compile(
-  "^LineaRollup(?:AlphaV3)? deployed: address=(0x[0-9a-fA-F]{40}) blockNumber=(\\d+)"
+  "^LineaRollupV(?:[0-9]+)? deployed: address=(0x[0-9a-fA-F]{40}) blockNumber=(\\d+)"
```

Comment:
Can we have version like V5.1 ? If yes this regex won't work if I am not wrong :)

Reply:
the contract itself won't be able to be named V5.x just the field inside (contract names don't like the `.`s)

Reply:
Ah yes you're right :)

Reply:
That's a good point. Let me check it out